miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/rest/PrinterRestController.java

```diff
@@ -0,0 +1,126 @@
+package uk.ac.bbsrc.tgac.miso.webapp.controller.rest;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.ws.rs.core.Response.Status;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.util.UriComponentsBuilder;
+
+import uk.ac.bbsrc.tgac.miso.core.data.Printer;
+import uk.ac.bbsrc.tgac.miso.core.util.PaginatedDataSource;
+import uk.ac.bbsrc.tgac.miso.dto.DataTablesResponseDto;
+import uk.ac.bbsrc.tgac.miso.dto.Dtos;
+import uk.ac.bbsrc.tgac.miso.dto.PrinterDto;
+import uk.ac.bbsrc.tgac.miso.service.PrinterService;
+
+@Controller
+@RequestMapping("/rest/printer")
+
+public class PrinterRestController extends RestController {
+  private static final Logger log = LoggerFactory.getLogger(PrinterRestController.class);
+  private final JQueryDataTableBackend<Printer, PrinterDto> jQueryBackend = new JQueryDataTableBackend<Printer, PrinterDto>() {
+
+    @Override
+    protected PrinterDto asDto(Printer model) {
+      return Dtos.asDto(model);
+    }
+
+    @Override
+    protected PaginatedDataSource<Printer> getSource() throws IOException {
+      return printerService;
+    }
+
+  };
+
+  @Autowired
+  private PrinterService printerService;
+
+  @RequestMapping(method = RequestMethod.POST, headers = { "Content-type=application/json" })
+  @ResponseBody
+  public PrinterDto create(@RequestBody PrinterDto dto) throws IOException {
+    Printer printer = Dtos.to(dto);
+    return get(printerService.create(printer));
+  }
+
+  @RequestMapping(value = "dt", method = RequestMethod.GET, produces = "application/json")
+  @ResponseBody
+  public DataTablesResponseDto<PrinterDto> dataTable(HttpServletRequest request,
+      HttpServletResponse response, UriComponentsBuilder uriBuilder) throws IOException {
+    return jQueryBackend.get(request, response, uriBuilder);
+  }
+
+  @RequestMapping(method = RequestMethod.DELETE, headers = { "Content-type=application/json" })
+  @ResponseBody
+  public ResponseEntity<?> delete(@RequestBody List<Long> printerIds) throws IOException {
```

Comment:
`@ResponseStatus(HttpStatus.NO_RESPONSE)`

Then you can change return type to void and remove the return statement